gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/x509/X509Config.groovy

```diff
@@ -180,4 +181,18 @@ class X509Config {
       http.securityContext().securityContextRepository(new X509SecurityContextRepository())
     }
   }
+
+  @ConditionalOnBean(IAPConfig)
+  @Bean
+  X509IAPConfig withIAPConfig() {
+    new X509IAPConfig()
+  }
+
+  class X509IAPConfig implements LdapSsoConfigurer {
```

Comment:
X509 + IAP implementing the LdapSso looks suspicious.

Reply:
Donez. these interfaces are all the same